{"instance_id": "UDMIS013EXT", "instance_filename": "UDMIS013EXT_40_N40_W1.0_R0.2.qubo.json", "solver_name": "QUBODwaveSolver", "args": ["1000", "None", "None"], "timestamps": {"start": 1702924482.62348, "end": 1702924488.93633}, "solver": {"exception": "", "traceback": "Traceback (most recent call last):\n  File \"C:\\Users\\admin\\Documents\\qopt\\benchmark\\bench.py\", line 152, in run\n    solver = self._run(instance_id, *args)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\admin\\Documents\\qopt\\benchmark\\bench_qubo.py\", line 138, in _run\n    response=response.to_serializable(),\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\ProgramData\\anaconda3\\envs\\qopt\\Lib\\site-packages\\dimod\\sampleset.py\", line 1779, in to_serializable\n    for name, data in self.data_vectors.items()}\n                      ^^^^^^^^^^^^^^^^^\n  File \"C:\\ProgramData\\anaconda3\\envs\\qopt\\Lib\\site-packages\\dimod\\sampleset.py\", line 1062, in data_vectors\n    return {field: self.record[field] for field in self.record.dtype.names\n                                                   ^^^^^^^^^^^\n  File \"C:\\ProgramData\\anaconda3\\envs\\qopt\\Lib\\site-packages\\dimod\\sampleset.py\", line 1121, in record\n    self.resolve()\n  File \"C:\\ProgramData\\anaconda3\\envs\\qopt\\Lib\\site-packages\\dimod\\sampleset.py\", line 1470, in resolve\n    samples = self._result_hook(self._future)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\ProgramData\\anaconda3\\envs\\qopt\\Lib\\site-packages\\dwave\\system\\composites\\embedding.py\", line 284, in async_unembed\n    sampleset = unembed_sampleset(response, embedding, source_bqm=bqm,\n                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\ProgramData\\anaconda3\\envs\\qopt\\Lib\\site-packages\\dwave\\embedding\\transforms.py\", line 626, in unembed_sampleset\n    record = target_sampleset.record\n             ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\ProgramData\\anaconda3\\envs\\qopt\\Lib\\site-packages\\dimod\\sampleset.py\", line 1121, in record\n    self.resolve()\n  File \"C:\\ProgramData\\anaconda3\\envs\\qopt\\Lib\\site-packages\\dimod\\sampleset.py\", line 1470, in resolve\n    samples = self._result_hook(self._future)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\ProgramData\\anaconda3\\envs\\qopt\\Lib\\site-packages\\dwave\\system\\samplers\\dwave_sampler.py\", line 440, in _hook\n    return resolve(computation)\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\ProgramData\\anaconda3\\envs\\qopt\\Lib\\site-packages\\dwave\\system\\samplers\\dwave_sampler.py\", line 430, in resolve\n    sampleset.resolve()\n  File \"C:\\ProgramData\\anaconda3\\envs\\qopt\\Lib\\site-packages\\dimod\\sampleset.py\", line 1470, in resolve\n    samples = self._result_hook(self._future)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\ProgramData\\anaconda3\\envs\\qopt\\Lib\\site-packages\\dwave\\cloud\\computation.py\", line 823, in <lambda>\n    self, lambda f: f.wait_sampleset())\n                    ^^^^^^^^^^^^^^^^^^\n  File \"C:\\ProgramData\\anaconda3\\envs\\qopt\\Lib\\site-packages\\dwave\\cloud\\computation.py\", line 755, in wait_sampleset\n    result = self._load_result()\n             ^^^^^^^^^^^^^^^^^^^\n  File \"C:\\ProgramData\\anaconda3\\envs\\qopt\\Lib\\site-packages\\dwave\\cloud\\computation.py\", line 900, in _load_result\n    self._decode()\n  File \"C:\\ProgramData\\anaconda3\\envs\\qopt\\Lib\\site-packages\\dwave\\cloud\\computation.py\", line 920, in _decode\n    self._result = self.solver.decode_response(self._message)\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\ProgramData\\anaconda3\\envs\\qopt\\Lib\\site-packages\\dwave\\cloud\\solver.py\", line 194, in decode_response\n    return self._decode_qp(msg)\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\ProgramData\\anaconda3\\envs\\qopt\\Lib\\site-packages\\dwave\\cloud\\solver.py\", line 181, in _decode_qp\n    return decode_qp_numpy(msg, return_matrix=self.return_matrix)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\ProgramData\\anaconda3\\envs\\qopt\\Lib\\site-packages\\dwave\\cloud\\coders.py\", line 282, in decode_qp_numpy\n    result['solutions'] = result['solutions'].tolist()\n                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nMemoryError\n", "fail_timestamp": 1702924488.920737}, "success": false, "version": "2.0.2"}